README.rst

```diff
@@ -328,3 +331,7 @@ POST Polygon
         [75, 29]
       ]
     }'
+
+With the "format_out" as "geojson" or "polygon", the result will be simple multipoints with the elevation values as 3rd dimension in the coordinates.
+
+With the "format_out" as "colorpolygon", the result will be a feature collection with the elevation values assigned as properties to geometries of polygon or multipolygon.
```

Comment:
Is it necessary to have polygon and colorpolygon? After testing both queries, the 'colorpolygon' query is the most appropriate. The 'polygon' query could be removed.

Reply:
OK